### PR TITLE
content: work on donation page

### PR DESCRIPTION
**Pull request by alexgarel**
This is a rework on the donation page (warning: on preprod, it will be visible only as donate/en.html, until we build redirections in docker nginx conf).

Rendering:

[small](https://user-images.githubusercontent.com/144788/143919044-da483843-191b-4b71-a62e-164b9f2a4cbf.png)

[medium](https://user-images.githubusercontent.com/144788/143919043-6a6f933d-6b02-42f5-9f7d-bfea22a36e28.png)

[large](https://user-images.githubusercontent.com/144788/143919040-b8958993-bb50-44fe-b0e4-2ece25b91e4a.png)

### CODE REVIEW

**Comment by stephanegigandet** [comment]
deployed on https://world.openfoodfacts.dev/donate-to-open-food-facts

the image paths were broken, I'll commit a fix

**Review by stephanegigandet — APPROVED** [review]
Looks good to me, I made minor text corrections directly on the branch, hope you don't mind.

File: html/donate/en.html [hunk 1]
```diff
@@ -132,22 +137,19 @@ <h3>Help us to fund the Open Food Facts 2022 budget!</h3>
...
-<h3>Help us to fund the Open Food Facts 2022 budget!</h3>
-<p>Open Food Facts is 100% free and independent of the food industry. We need your help to continue and to grow the project.</p>
-</div>
-
-        <img
-          src="https://static.openfoodfacts.org/images/svg/crowdsourcing-icon.svg"
-          style="width:240px;float:left;margin-right:5px;"
-          alt="Yes we scan! logo"
-        />
+        <div class="panel callout" style="margin-top:0.5rem">
```
Comment (CharlesNepote) [inline html/donate/en.html:146]:
```suggestion
        <div class="panel callout" style="margin-top:0.5rem;padding-bottom:0.5em;">
```

File: html/donate/en.html [hunk 2]
```diff
@@ -132,22 +137,19 @@ <h3>Help us to fund the Open Food Facts 2022 budget!</h3>
...
+          <h3>Help us to fund the Open Food Facts 2022 budget!</h3>
```
Comment (CharlesNepote) [inline html/donate/en.html:147]:
```suggestion
          <p><strong>Help us to fund the Open Food Facts 2022 budget!</strong></p>
```

**Review by CharlesNepote — CHANGES_REQUESTED** [review]

**Review by stephanegigandet — APPROVED** [review]

File: html/donate/en.html (hunk 2 above)
Comment (stephanegigandet) [reply html/donate/en.html:149]: The <p><strong> actually takes more vertical space on my screen.

File: html/donate/en.html (hunk 1 above)
Comment (stephanegigandet) [reply html/donate/en.html:148]: The padding becomes asymetric, it feelds weird. We reduced the text quite a lot already, let's see how it looks now.

**Review by CharlesNepote — APPROVED** [review]

**Pull request merged**